### PR TITLE
use new endpoint to read SKUs during validation and updates for backu…

### PR DESCRIPTION
**Pull request by yuchouMS**
…p container uri

## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above. https://github.com/Azure/azure-rest-api-specs/pull/1334
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by yuchouMS** [comment]
PR for backup container uri swagger spec changes: https://github.com/Azure/azure-rest-api-specs/pull/1326

**Comment by hovsepm** [comment]
@azuresdkci test this please

**Comment by yuchouMS** [comment]
@hovsepm could you help review the pr? thanks

**Comment by shahabhijeet** [comment]
@yuchouMS could you sign the CLA. https://cla.microsoft.com.
If you are MS employee, please connect your id with GitHub id at this location
https://opensource.microsoft.com/link

**Comment by yuchouMS** [comment]
@shahabhijeet I've already connected my id with Github long time ago. but when I click the URL it shows: "opensource.microsoft.com is currently unable to handle this request. HTTP ERROR 500." something wrong with the website?

File: src/SDKs/AnalysisServices/Management.Analysis/Generated/Models/SkuEnumerationForNewResourceResult.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+// 
+// Code generated by Microsoft (R) AutoRest Code Generator 0.17.0.0
```
Comment (shahabhijeet) [inline src/SDKs/AnalysisServices/Management.Analysis/Generated/Models/SkuEnumerationForNewResourceResult.cs]: @yuchouMS please use latest autoRest to generate your SDK.

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

**Comment by shahabhijeet** [comment]
@azuresdkci add to whitelist

File: src/SDKs/AnalysisServices/Management.Analysis/Generated/Models/SkuEnumerationForNewResourceResult.cs (hunk 1 above)
Comment (yuchouMS) [inline src/SDKs/AnalysisServices/Management.Analysis/Generated/Models/SkuEnumerationForNewResourceResult.cs]: @shahabhijeet fixed, please take another look. thanks

**Comment by yuchouMS** [comment]
@shahabhijeet please help review when you have time, thanks

**Comment by shahabhijeet** [comment]
@yuchouMS you will need major version bump for your nuget as well as AssemblyFileVersion/AssemblyVersion bump.

**Comment by yuchouMS** [comment]
@shahabhijeet fixed, please help review. thanks

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**